src/javascript/binary/pages/trade/starttime.js

```diff
@@ -20,11 +20,15 @@ function displayStartDates() {
 
     var startDates = Contract.startDates();
 
-    if (startDates) {
+    if (startDates && startDates.length && sessionStorage.formname !== 'higherlower') {
```

Comment:
we should not have form specific here. if startDates is not there it should be fine